src/box/memtx_space.c

```diff
@@ -1073,6 +1073,13 @@ memtx_space_check_format(struct space *space, struct tuple_format *format)
 
 		ERROR_INJECT_YIELD(ERRINJ_CHECK_FORMAT_DELAY);
 
+		ERROR_INJECT_COUNTDOWN(ERRINJ_CHECK_FORMAT_DELAY_COUNTDOWN, {
+			struct errinj *e =
+				errinj(ERRINJ_CHECK_FORMAT_DELAY, ERRINJ_BOOL);
+			e->bparam = true;
+			ERROR_INJECT_YIELD(ERRINJ_CHECK_FORMAT_DELAY);
+		});
```

Comment:
If you switch these 2 ERROR_INJECT calls, then you can drop the one that you have on line 1080.

Reply:
How will this work, if one wants to enable `ERRINJ_CHECK_FORMAT_DELAY` independently of `ERRINJ_CHECK_FORMAT_DELAY_COUNTDOWN`? For instance, if the latter is set to `-1` (default).

Also refer to `ERRINJ_WAL_DELAY{_COUNTDOWN}`?

Reply:
I guess we will ignore this for now.

Reply:
Sorry, I forgot to push the fix.